node_modules/gh-admins/lib/rest.js

```diff
@@ -53,7 +55,13 @@ GrassHopper.globalAdminRouter.on('get', '/api/admins', function(req, res) {
  * @Return      {Me}                                            The current global administrator
  */
 GrassHopper.globalAdminRouter.on('get', '/api/me', function(req, res) {
-    res.status(200).send(req.ctx.user);
+    var me = {
```

Comment:
Thinking about this more, it would be good to add returning the me object after successful login to this PR